openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/CEEnvironmentProcessor.java

```diff
@@ -263,4 +264,13 @@ private void delay(OpenShiftAdapter client, final List<? extends OpenShiftResour
     private void additionalCleanup(OpenShiftAdapter client, Map<String, String> labels) throws Exception {
         client.cleanRemnants(labels);
     }
+
+    private void createUniqueNamespace(CubeOpenShiftConfiguration configuration, String uniqueKey) {
+        namespace = configuration.getNamespace() + "-" + uniqueKey.toLowerCase().replaceAll("[^a-z0-9-]", "-");
+        if (namespace.chars().count() > 63) {
```

Comment:
While creating namespace for method, it'll be good if we think for class name as well, As two classes can have same method name. Also we should  log it to avoid confusion of user about from which class this method belongs to.

namespace = `configuration.getNamespace() + "_" +  testClass + "_" + methodName`. 

If it has more than 63 characters then we should fall back to some readable name(e.g. `configuration.getNamespace() + randomUniqueId`) and log it like "fallback to "itest_5876667" as `namespace-testclass-methodName` is greater than 63 characters, so user knows the reason about trimming.